src/Entity/Contract.php

```diff
@@ -91,6 +103,16 @@ public function getId()
         return $this->id;
     }
 
+    public function getContractor()
+    {
+        if (count($this->restaurants)) {
+            return $this->restaurants[0];
+        } else if (count($this->businessRestaurantGroups)) {
+            return $this->businessRestaurantGroups[0];
```

Comment:
It's more safe to use `array_shift($array)` than `$array[0]`

Reply:
yes OK i ll change
the base problem is that  we don't have enough check on Contract objects

https://github.com/coopcycle/coopcycle-web/issues/4254